codes/20057_py3_sjskgks.py

```diff
@@ -0,0 +1,67 @@
+import sys
+input = sys.stdin.readline
+
+sizeOfsand = int(input())
+sand = []
+for _ in range(sizeOfsand):
+    sand.append(list(map(int, input().split())))
+    
+def Next_pos(direc, cur_row, cur_col):
+    if direc=='left':
+        return cur_row, cur_col-1
+    elif direc=='down':
+        return cur_row+1, cur_col
+    elif direc=='right':
+        return cur_row, cur_col+1
+    else:
+        return cur_row-1, cur_col
+
+def Next_move(cur_direction, numOfmove):
+    if cur_direction=='left':
+        return ['down', numOfmove]
+    elif cur_direction=='down':
+        return ['right', numOfmove+1]
+    elif cur_direction=='right':
+        return ['up', numOfmove]
+    else:
+        return ['left', numOfmove+1]
+    
+def Scatter(row, col): # 공격받은 위치

```

Comment:
여기도요!
```suggestion
def scatter(row, col): # 공격받은 위치
```
그리고 scatter함수의 논리적 의미가 궁금합니다. 지금은 단순히 %를 계산하는데 함수로 빼야할지 의문입니다. 혹시 reserve 때문이신가요?

Reply:
다른 이유는 없고 그냥 깔끔할거같아서 했는데 생각해보니 굳이 뺄필요는 없었던거같네용 헤헷,, 감사합니당